subsys/bluetooth/services/nus.c

```diff
@@ -35,17 +33,9 @@ static bool is_notification_enabled(struct bt_conn *conn,
 	return false;
 }
 
-static void nuslc_ccc_cfg_changed(const struct bt_gatt_attr *attr,
-				  u16_t value)
+static void nus_ccc_changed(const struct bt_gatt_attr *attr, u16_t value)
 {
-	char addr[BT_ADDR_LE_STR_LEN] = {0};
-	struct bt_gatt_ccc_cfg *cfg =
-		(struct bt_gatt_ccc_cfg *)attr->user_data;
-
-	bt_addr_le_to_str(&cfg->peer, addr, ARRAY_SIZE(addr));
-
-	LOG_DBG("%s CCCD has changed, state: %s", log_strdup(addr),
-		value ? "notification enabled" : "notification disabled");
+	LOG_DBG("value 0x%04x", value);
```

Comment:
Why we cannot print which device changed the cccd. Does `attr` not contains this data now ?

Reply:
@KAGA164 No, we don't know which one it was.
We have to add ccc_write callback instead. If you want I can add it.

Reply:
Ok. Do as you prefer. In this case we needs this callback ? It only print the  current value of the CCCD.

Reply:
I'll just remove it. No need to add the other one either. Since it would only prints information anyway.